zookeeper/server/init.sls

```diff
@@ -86,6 +88,8 @@ zookeeper-service:
     - template: jinja
     - context:
       alt_home: {{ zk.alt_home }}
+      user: {{ zk.user }}
+      group: {{ zk.group }}
```

Comment:
The `group` variable is redundant here though, but it's OK. May become useful in the future.